setup.py

```diff
@@ -4,7 +4,6 @@
 
 config = read_configuration('setup.cfg')
 version = config['metadata']['version']
-dbt_version = version.rsplit('.', 1)[0]
-install_requires = [f'dbt-core~={dbt_version}', config['options']['install_requires']]
+install_requires = [config['options']['install_requires']]
```

Comment:
Is this supposed to be a part of this PR?

Reply:
No, that somehow got copied over from a different PR. Fixed.